### PR TITLE
data: prrajput1199

### PR DESCRIPTION
**Pull request by prrajput1199**

### CODE REVIEW

File: data/prrajput1199.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open-Source enthusiastic",
+  "avatar": "https://github.com/prrajput1199.png"
+  "links":[
+  {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/PratikR61035031",
+      "icon": "FaTwitter"
+  }
```
Comment (senali-d) [inline data/prrajput1199.json:12]: Please add `,` after the `}`
```suggestion
  },
```

File: data/prrajput1199.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  {
+    "name": "Follow me on Github",
+    "url" :"https://github.com/prrajput1199",
+    "icon": "Fagithub"
+  }
```
Comment (senali-d) [inline data/prrajput1199.json:17]: Please add `,` after the `}`
```suggestion
  },
```

**Review by senali-d — CHANGES_REQUESTED** [review]

**Comment by prrajput1199** [comment]
@senali-d I have made the changes you suggested. Looking forward for your further review.

**Review by iamthecloverly — APPROVED** [review]

**Review by Sativarsainath-26 — APPROVED** [review]
Looks Good :+1:

**Comment by Sativarsainath-26** [comment]
Thanks for contributing :star2:

**Review by lewisushindi — DISMISSED** [review]

File: data/prrajput1199.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Pratik Rajput",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open-Source enthusiastic",
+  "avatar": "https://github.com/prrajput1199.png"
```
Comment (devvsakib) [inline data/prrajput1199.json:6]: Hi, you forget to add tags
example:
  "tags": ["Open Source", "React"],

image

**Review by devvsakib — CHANGES_REQUESTED** [review]
Hi, you forget to add tags
example:
  "tags": ["Open Source", "React"],

image

File: data/prrajput1199.json (hunk 3 above)
Comment (prrajput1199) [reply data/prrajput1199.json:6]: @devvsakib can you tell me where I should add tags? I am unable to understand.

Comment (devvsakib) [reply data/prrajput1199.json:6]: @prrajput1199  sure, 
this way you can add your tags
image

Comment (prrajput1199) [reply data/prrajput1199.json]: @devvsakib Can you check now?? I have made the changes

**Review by devvsakib — APPROVED** [review]
PERFECT. good to merge

File: data/prrajput1199.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  "links":[
+  {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/PratikR61035031",
+      "icon": "FaTwitter"
+  },
+  {
+    "name": "Follow me on Github",
+    "url" :"https://github.com/prrajput1199",
+    "icon": "Fagithub"
```
Comment (kumarsonsoff3) [inline data/prrajput1199.json:17]: This is an invalid icon. Please update it as suggested! 

```suggestion
    "icon": "FaGithub"
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Comment by prrajput1199** [comment]
@kumarsonsoff3 I have made the changes you suggested.

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good & great collaboration everyone! 🎉 

<p align="center"></p>

**Pull request merged**